CONTRIBUTING.md

```diff
@@ -82,3 +81,26 @@ Before a pull request can be merged:
 * The feature branch must be rebased so it contains the latest changes from the target branch
 * The CI has to pass successfully
 * Every comment has to be addressed and resolved
+
+## Testing
+
+Tests are written using the *[Ginkgo](https://onsi.github.io/ginkgo/)* framework and here are some general advices when writing tests for the `release-service`:
+
+* Start the descriptions of `It` blocks in lowercase and try to be as more descriptive as possible.
+* Avoid using `_` for errors;
+* Only create `..._suite_test.go` files if your test setup is big enough to justify it; IE: has more than the test suite name and the teardown;
+* When required, remember to add the `CRD`'s at the `envtest` setup, for instance: [release-service/release_suite_test.go at main · redhat-appstudio/release-service · GitHub](https://github.com/redhat-appstudio/release-service/blob/main/controllers/release/release_suite_test.go#L65) - remembering this saves a lot of time;
+* After `Create()` or `Update()` objects, use `Get()` before making assurances as the object might be outdated;
+* Some assurances are likely to require usage of `Eventually` blocks instead of or in addition to `Expect` blocks;
+
+### Useful links:
+
+Links that may be used as a starting point:
+
+* https://onsi.github.io/ginkgo/#getting-started
+* [gomega package - github.com/onsi/gomega - Go Packages](https://pkg.go.dev/github.com/onsi/gomega#section-readme)
+
```

Comment:
Add the following links:

- [Writting controller tests](https://book.kubebuilder.io/cronjob-tutorial/writing-tests.html)
- [Testing your Operator project](https://master.sdk.operatorframework.io/docs/building-operators/golang/testing/)